crates/modelardb_server/src/configuration.rs

```diff
@@ -129,11 +137,11 @@ mod tests {
     use std::path::Path;
     use std::sync::Arc;
 
+    use modelardb_common::metadata::try_new_sqlite_table_metadata_manager;
```

Comment:
Maybe only import `modelardb_common::metadata` and use the function as  `metadata::try_new_sqlite_table_metadata_manager()` for consistency.

Reply:
Fixed.